alf/examples/muzero_atari_conf.py

```diff
@@ -411,6 +411,7 @@ def _scale_grad(scale):
     "MCTSAlgorithm",
     num_simulations=52,
     num_parallel_sims=2,
+    discount=discount,
```

Comment:
Is this automatically set by MuzeroAlgorithm and MuzeroRepresentationLearner?

Reply:
It used to be, and is still automatically set by `MuzeroAlgorithm`, but not by `MuzeroRepresentationLearner`. Since we want to support algorithm such as `PPOAlgorithm` as reanalyze algorithm, and they do not accept `discount` as argument, it is easier to set them specifically in configuration for `MCTSAlgorithm`.